docs/xdr/features/collect/integrations/endpoint/microsoft_intune.md

```diff
@@ -0,0 +1,41 @@
+uuid: c20528c1-621e-4959-83ba-652eca2e8ed0
+name: Microsoft Intune
+type: intake
+
+## Overview
+
+Microsoft Intune helps you protect your workforce's corporate data by managing devices and apps. Intune provides mobile device management (MDM) and mobile app management (MAM) from a secure cloud-based service that is administered using the Microsoft Endpoint Manager admin center. Using Intune, you ensure your workforce's corporate resources (data, devices, and apps) are correctly configured, accessed, and updated, meeting your company's compliance policies and requirements.
+
+This setup guide describe how to forward events produced by `Microsoft Intune` to SEKOIA.IO.
+
```

Comment:
Recent change on the documentation, to associate rule to datasource, we should include a new autogenerated document
```suggestion
{!_shared_content/operations_center/detection/generated/suggested_rules_c20528c1-621e-4959-83ba-652eca2e8ed0_do_not_edit_manually.md!}

```